pkg/services/store/entity/sqlstash/sql_storage_server.go

```diff
@@ -80,7 +76,7 @@ type sqlEntityServer struct {
 	ctx         context.Context // TODO: remove
 	cancel      context.CancelFunc
 	stream      chan *entity.EntityWatchResponse
-	tracer      tracing.Tracer
+	tracer      trace.Tracer
```

Comment:
Easier to fix tests, `tracing.Tracer` is the same as `trace.Tracer` but with an additional method to inject HTTP headers (which we don't need here since it's a gRPC service), and there isn't a noop implementation of `tracing.Tracer`.